app/helpers/katello/hosts_and_hostgroups_helper.rb

```diff
@@ -91,26 +91,48 @@ def fetch_content_view(host, options = {})
       selected_content_view if selected_content_view.present?
     end
 
+    def accessible_lifecycle_environments(org, host)
+      selected = lifecycle_environment(host)
+      envs = org.kt_environments.readable
+      envs |= [selected] if selected.present? && org == selected.organization
+      envs
+    end
+
+    def accessible_content_proxies(obj)
```

Comment:
i think the reason its 'obj' is that it could be a host or hostgroup, happy to rename it to host_or_hostgroup or something like that.  In some of our code (what wasn't pulled in from foreman like this one) we label it just as 'host', but i think that may cause more confusion than just 'obj'.